channels/ro.m3u

```diff
@@ -68,6 +68,8 @@ https://rtmp.digitalbroadcast.ro/moozhits/moozhits.m3u8
 https://rtmp.digitalbroadcast.ro/moozro/moozro.m3u8
 #EXTINF:-1 tvg-id="MusicChannelRomania.ro" tvg-country="RO" tvg-language="Romanian" tvg-logo="https://i.imgur.com/PRgvj4c.png" group-title="Music",Music Channel Romania (576p) [Offline]
 https://edge126.rcs-rds.ro/utvedge/musicchannelhq.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="Mireasa.ro" tvg-country="RO" tvg-language="Romanian" tvg-logo="" group-title="",Mireasa
+https://iptv-all.lanesh4d0w.codes/romania/mireasa
```

Comment:
Black screen

Reply:
Yup, same on AntenaPlay it seems, I smell the Not 24/7.

Reply:
It seems it's a real-time reality show stream, that might explain why, linked to Antena 1. Certainly not 24/7.

Reply:
Be nice to get that AntenaSport https://as.ro/

Reply:
See last addition =)